src/oop/creating_objects_from_classes/hard/q2/QuestionText.md

```diff
@@ -0,0 +1,48 @@
+# oop-creating_objects_from_classes-hard-q2
+
+## Question Text
```

Comment:
Is there a reason for the two different question texts?

Reply:
Sorry, copy paste issue I evidently forgot to clean up